test/suites/standard/hostname.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Documentation       Tests verifying hostname resolution
+
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/oc.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           restart    slow
+
+
+*** Variables ***
+${NEW_HOSTNAME}     microshift.local
+${OLD_HOSTNAME}     ${EMPTY}
+
+
+*** Test Cases ***
+Verify local name resolution
```

Comment:
it would be more descriptive  to include mdns in the  test name.

Reply:
Was trying to refer more to the local domain the hostname has in this test. The mDNS detail is within the documentation part. WDYT?

Reply:
well adding `.local` to the domain will trigger chain of events in the cluster (exposing mDNS service ie), if we only wants to test the end result (.local resolution) that its fine as it is.